nservicebus/tools/migrate-to-native-delivery.md

```diff
@@ -4,6 +4,8 @@ summary: An overview of the tool supporting migrating from timeout manager to na
 reviewed: 2020-06-22
 ---
 
+NOTE: Tools are designed for single or occasional use. Only the latest version of a given tool is supported. Users should consider upgrading to the latest version of a tool before each use.
+
```

Comment:
Not sure this was meant to be here?

Reply:
I copied that note from https://docs.particular.net/nservicebus/tools/, because this page only has a single tool listed and in order to not remove the need to have that page in the navigation.